etc/cas.properties

```diff
@@ -51,6 +51,11 @@ cas.okstate.login.url=https://stgcas.okstate.edu/cas/login
 cas.okstate.client.name=okstate
 cas.okstate.cas.protocol=SAML
 #
+# CAS: Ferris State University
+cas.ferris.login.url=https://lpcas.ferris.edu/cas-web/login
```

Comment:
Update: need to update this on each `env`, and in the future we should load CAS client from JSON only on needed requirement (`prod` and `test`).